packages/blade/src/components/Checkbox/checkboxTokens.ts

```diff
@@ -86,12 +86,12 @@ const checkboxIconColors: CheckboxIconColors = {
 const checkboxHoverTokens: SelectorInputHoverTokens = {
   default: {
     background: {
-      checked: 'colors.brand.primary.600',
-      unchecked: 'colors.brand.gray.a50.lowContrast',
+      checked: 'colors.interactive.background.primary.highlighted',
+      unchecked: 'colors.interactive.background.gray.faded',
     },
     border: {
-      checked: 'colors.brand.primary.600',
-      unchecked: 'colors.brand.gray.500.lowContrast',
+      checked: 'colors.interactive.border.background.highlighted', // Intentionally not using border tokens here since we want to match the background color
```

Comment:
wrong token? 

border.background?

Reply:
Intentional

Reply:
I'm saying this token itself is wrong. It doesn't exist. 

<img width="985" alt="image" src="https://github.com/razorpay/blade/assets/35374649/46f70581-cfce-48c4-ab8e-57db784540a0">